src/content_handlers/apt_handler/src/apt_handler.cpp

```diff
@@ -288,7 +273,7 @@ ADUC_Result AptHandlerImpl::Apply()
         return ADUC_Result{ ADUC_ApplyResult_Failure, ADUC_ERC_APT_HANDLER_INSTALLCRITERIA_PERSIST_FAILURE };
     }
 
-    if (g_restartRequiredAfterApplied)
+    if (_aptContent->AgentRestartRequired)
     {
         // Always require a agent restart after self updated.
         Log_Debug("The install task completed successfully, ADU Agent restart is required for this update.");
```

Comment:
comment is no longer correct.

Reply:
removed

---
In reply to: [596182512](https://github.com/Azure/iot-hub-device-update/pull/17#discussion_r596182512) [](ancestors = 596182512)